src/App.vue

```diff
@@ -644,17 +644,6 @@ export default {
       // set addresses
       this.storeAddresses({ data: application.offices || [] })
 
-      // Format party roles from IA
-      application.parties.forEach( // Check each party for roles
-        party => party.roles?.forEach( // Check each role for roleType
-          (role, index) => {
-            if (role.roleType) { // If roleType exists, assign it to parent role index
-              party.roles[index] = role.roleType.toLowerCase()
-            }
-          }
-        )
-      )
```

Comment:
No longer needed?

Reply:
I don't believe so no, but to verify, what state is this triggered. 
Is this after a user completes a Filed IA, that interm state before they have to reload there dashboard?

Reply:
Going to try and verify the structure in the IA. This could still be an edge case

Reply:
Ok, verified 👍

Reply:
Did you file an IA to test this?

Reply:
> I don't believe so no, but to verify, what state is this triggered. Is this after a user completes a Filed IA, that interm state before they have to reload there dashboard?

Yeah, looks like that scenario.

When the entity is opened using a permanent ID then the IA filing will naturally appear in the ledger.

Reply:
Exactly, you bet. The state where it’s still a temp reg but with a filing in the history.

Reply:
You da man.
💯 
😎